dll/win32/shimgvw/shimgvw.c

```diff
@@ -524,9 +524,10 @@ pLoadImageFromNode(SHIMGVW_FILENODE *node, HWND hwnd)
     pLoadImage(node->FileName);
 
     LoadStringW(hInstance, IDS_APPTITLE, szResStr, _countof(szResStr));
-    if (image != NULL)
+
+    pchFileTitle = PathFindFileNameW(node->FileName);
+    if (pchFileTitle != NULL && wcsicmp(pchFileTitle, L"")) 
```

Comment:
```suggestion
    if (pchFileTitle && wcsicmp(pchFileTitle, L"")) 
```

Reply:
This isn’t necessary, fix it.

Reply:
What part is not necessary?
Also, the construct `wcsicmp(pchFileTitle, L"")` can be more efficiently replaced by `*pchFileTitle` .

Reply:
"!= NULL" isn't necessary.

Change `wcsicmp(pchFileTitle, L"")` to `*pchFileTitle`.